extension/app/html/addressBook.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html style = 'background-color: var(--bg-color); overflow-y: inherit;'>
+	<head>
+		<meta charset = 'utf-8'>
+		<link rel = 'icon' type = 'image/x-icon' href = 'favicon.ico'>
+	</head>
+	<body style = 'background-color: var(--bg-color); margin: auto;'>
+		<meta name = 'viewport' content = 'width = device-width, initial-scale = 1' />
+		<link rel = 'stylesheet' type = 'text/css' href = '../css/bulma.css' />
+		<link rel = 'stylesheet' type = 'text/css' href = '../css/bulma-divider.css' />
+		<link rel = 'stylesheet' type = 'text/css' href = '../css/interceptor.css' />
+		<script src = '../vendor/webextension-polyfill/browser-polyfill.js'></script>
+		<script type = 'importmap'>
+		{
+			"imports": {
+				"ethers": "../vendor/ethers/ethers.esm.js",
+				"webextension-polyfill": "../vendor/webextension-polyfill/browser-polyfill.js",
+				"preact": "../vendor/preact/preact.module.js",
+				"preact/jsx-runtime": "../vendor/preact/jsx-runtime/jsxRuntime.module.js",
+				"preact/hooks": "../vendor/preact/hooks/hooks.module.js",
+				"funtypes": "../vendor/funtypes/index.mjs",
+				"node-fetch": "../vendor/node-fetch/index.mjs",
+				"@zoltu/ethereum-abi-encoder": "../vendor/@zoltu/ethereum-abi-encoder/index.js",
+				"@zoltu/ethereum-crypto": "../vendor/@zoltu/ethereum-crypto/index.js",
```

Comment:
We should consider moving away from `@zoltu/ethereum-crypto`.  There may also be better alternatives to rlp-encoder and abi-encoder as well that could be worth investigating.

Reply:
created an issue to look into that